src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -506,6 +540,17 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
         )
     }
 
+    private fun enableProductsAccordingToServerSetting() {
+        pluginSettings().apply {
+            sastOnServerEnabled = service<SnykApiService>().sastOnServerEnabled
+            ossScanEnable = true
+            advisorEnable = sastOnServerEnabled!!
+            iacScanEnabled = isIacEnabled() && sastOnServerEnabled!!
```

Comment:
how iacScanEnabled and sastOnServerEnabled are connected?

Reply:
Only access to Advisor and IaC if non-free customer - I talked with Georgi about that, and he was okay with it. But maybe we should re-verify the product strategy on that, to be sure.

Reply:
Just clarified with Georgi. They are not. Fix is pushed.